sdk/freertos_app_cpu0/src/usr/game/task_game.h

```diff
@@ -0,0 +1,48 @@
+#ifndef TASK_GAME_H
+#define TASK_GAME_H
+
+/* FreeRTOS includes */
+#include "FreeRTOS.h"
+#include "task.h"
+
+// Frequency that this task is called (in Hz)
+//
+// Must be less than or equal to scheduler updates per second
+// This value is defined in sys/scheduler.h and defaults to 10kHz.
+// Note that it can be overridden via usr/user_config.h
+#define TASK_GAME_UPDATES_PER_SEC (5)
+
+// Microseconds interval between when task is called
+//
+// This is what scheduler actually uses to run task,
+// but is generated via define above
+#define TASK_GAME_INTERVAL_USEC (USEC_IN_SEC / TASK_GAME_UPDATES_PER_SEC)
```

Comment:
I noticed your game task has a different way of defining its `interval_usec` stuff versus the other system tasks.

Eventually, we should be consistent on the "recommended" way of creating tasks.

If we want your game to be kept in the codebase for the full `v2` release, we'll need to update this. Otherwise, if the game will get removed, it does not matter

Reply:
Yeah, do we need the "game" in the codebase? This could become confusing clutter in the future if it's not a required resource.